docs/BreakingChanges/075 TargetFrameworkName for default app domain no longer defaults to null if not set.md

```diff
@@ -19,7 +19,7 @@ The TargetFrameworkName was previously null in the default app domain, unless it
 Code should be updated to not depend on `AppDomainSetup.TargetFrameworkName` defaulting to null. If it is required that this property continue to evaluate to null, it can be explicitly set to that value.
 
 ### Affected APIs
-* `M:System.AppDomainSetup.get_TargetFrameworkName`
+* `P:System.AppDomainSetup.TargetFrameworkName`
```

Comment:
Ah, I hadn't considered these from the perspective of auto-generating MSDN docs. Yea, if we can map accessors to properties for purposes of doc generation, that would be ideal. If not, making them properties (and adding a little bit of noise to reports) is probably acceptable in the name of getting docs auto-generated.